packages/icons/scripts/build.js

```diff
@@ -12,42 +12,67 @@ const generate = require("@babel/generator").default;
 const SVGO = require("svgo");
 const svgstore = require("svgstore");
 const iconSpriteConfig = require("../iconSpriteConfig.js");
-const svgo = new SVGO({
-  plugins: [
-    {
-      removeDoctype: true,
-    },
-    {
-      cleanupIDs: false,
-    }
-  ]
-});
+const gradientsToDefs = require("./gradientsToDefs");
+const instantiateSVGO = removeFill =>
+  new SVGO({
+    plugins: [
+      {
+        removeDoctype: true
+      },
+      {
+        cleanupIDs: false
+      },
+      {
+        removeUnknownsAndDefaults: true
+      },
+      removeFill
+        ? {
+            removeAttrs: { attrs: ["fill"] }
+          }
+        : {}
+    ]
+  });
 const buildDirPath = path.join(__dirname, "../", "dist");
-const distDirPath = path.join(__dirname, "../../../", "dist", "packages", "icons", "dist");
+const distDirPath = path.join(
+  __dirname,
+  "../../../",
+  "dist",
+  "packages",
+  "icons",
+  "dist"
+);
 
 const getFilePaths = dir =>
-  readdirSync(dir).map(file => `${dir}/${file}`);
+  readdirSync(dir)
+    .map(file => {
+      return `${dir}/${file}`;
+    })
+    .filter(file => file.includes(".svg"));
```

Comment:
💡  Even though I don't think that will ever be an issue, but an `file.endsWith` is probably be better?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith